README.md

```diff
@@ -1 +1,107 @@
+
+## Documentation of pipeline:
+
+See all the [details of running a FAH server and setting up new projects first](https://github.com/foldingathome/fah-docs/wiki).
+
+- [ ] First, the project manager sets up one or more protein PDB files and ligand SDF files
+- [ ] We select a FAH complex project ID (`complex_projid`) and solvent project ID (`solvent_projid`) pair from the [reserved set of projects wiki](https://github.com/FoldingAtHome/fah-docs/wiki/Project-numbers). FAH is limited to 65536 projects over its entire lifetime, so we have to conserve project numbers when possible. The complex project will have a large number of total atoms (~80K) and the solvent project will have a small number (~4K) where we can keep adding `RUN`s for new systems or new transformations as long as they fit in the simulation boxes.
+- [ ] When we want to add one or more new ligands to the set (e.g. by adding rows to the [COVID Moonshot Compound Tracker](https://covid.postera.ai/covid/submissions/compounds), which can be [downloaded via the "Export CSV" button](https://covid.postera.ai/covid/submissions.csv) or at the associated [master GitHub repo](https://github.com/postera-ai/COVID_moonshot_submissions) in a [CSV file](https://github.com/postera-ai/COVID_moonshot_submissions/blob/master/covid_submissions_all_info.csv)), we convert the SMILES for the ligand into actual 3D molecules, expanding stereochemistry and store in SDF, and then identify which reference protein PDB structures we will create alchemical transformations (one transformation per FAH `RUN` for).  [JDC will provide script for this part, still being worked out.]
+Hannah currently uses a [script](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/create-json.py) to populate a [JSON file](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/aminopyridines%202020-07-04.json), but we don't need to use this format (unless Hannah says otherwise).
+- [ ] Hannah uses a script to parallelize the setup of individual transformations in parallel using our cluster (`lilac`) via an [LSF bash script](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/submit-ligpairs.sh) and a [python `run.py`](https://github.com/choderalab/PersesAtFoldingAtHome/blob/master/SARS-CoV-2-Mpro/run.py).
+- [ ] To set up a single ligand transformation (`i -> j`), we run the `perses` command-line entry point `perses-fah config.yaml`, where `config.yaml` looks like this:
+```yaml
+protein_pdb: CDK2_protein.pdb
+ligand_file: CDK2_ligands_shifted.sdf
+old_ligand_index: 0
+new_ligand_index: 1
+forcefield_files:
+    - amber/ff14SB.xml # ff14SB protein force field
+    - amber/tip3p_standard.xml # TIP3P and recommended monovalent ion parameters
+    - amber/tip3p_HFE_multivalent.xml # for divalent ions
+    - amber/phosaa10.xml # HANDLES THE TPO
+atom_expression:
+    - IntType
+bond_expression:
+    - DefaultBonds
+complex_projid: 13418
+solvent_projid: 13419
```

Comment:
They have been so far, but they don't need to be. It just makes sense because we are setting them up at the same time, and they are different simulation sizes so can't be in the same project ID

Reply:
Thanks! Also, I think we need `trajectory_directory` here? Like https://github.com/choderalab/perses/commit/1d8b550d7c632f77f37728dc4f98e16e364faa0f